### ISSUE TITLE
Not working on Windows

### ISSUE THREAD

**Issue by abbaselmas** [nbedos/termtosvg#13]
import fcntl
ModuleNotFoundError: No module named 'fcntl'

**Comment by nbedos** [comment]
Do you run termtosvg inside Windows Subsystem for Linux ?

**Comment by abbaselmas** [comment]
no. directly from windows command prompt!
does termtosvg not supporting windows yet?

**Comment by nbedos** [comment]
No, sorry, there is no support for Windows. termtosvg might work inside Windows Subsystem for Linux, but it has not been tested.

**Comment by okonomiyaki3000** [comment]
There is no fcntl for Windows. It seems the only thing fcntl is used for here is to set the size of the terminal. Supposedly, the substitutes (sometimes) for fcntl on windows are win32api calls (https://stackoverflow.com/questions/1422368/fcntl-substitute-on-windows#1422436) so maybe there would be a way to use them to do this thing. I'm not actually familiar with it at all though, just the result of some quick searching.

**Comment by bacongobbler** [comment]
Can confirm that termtosvg works on Windows Subsystem for Linux. However, can we keep this ticket open to track Windows support for termtosvg? I've been poking and prodding at the win32 api calls to see if I can make termtosvg supported on Windows. Thanks!

**Comment by nbedos** [comment]
Thanks for the interest. I've reopened this issue, but I can't guarantee support for Windows will be added to termtosvg.

@bacongobbler: `$Host.UI.RawUI.GetBufferContents` seems to be the way to capture the screen of the terminal. Is that what you're looking into?

https://blogs.msdn.microsoft.com/powershell/2009/01/11/colorized-capture-of-console-screen-in-html-and-rtf/

**Comment by xindzju** [comment]
> Do you run termtosvg inside Windows Subsystem for Linux ?
It seems to not work inside windows Subsystem for Linux?
the prompts show as below pic:
![image](https://user-images.githubusercontent.com/42408511/45867219-8309fc00-bdb5-11e8-89d3-df0318871569.png)

Is there
 suggestion to fix it?
